Find a functionally equivalent code:android.widget.ShareActionProvider.setActivityChooserPolicyIfNeeded:COMMENT
Method Modifier: private     
Comment:/**
 * Set the activity chooser policy of the model backed by the current
 * share history file if needed which is if there is a registered callback.
 */

@@ -3,9 +3,9 @@
         return;
     }
     if (mOnChooseActivityListener == null) {
-        mOnChooseActivityListener = new ShareAcitivityChooserModelPolicy();
+        mOnChooseActivityListener = new ShareActivityChooserModelPolicy();
     }
     ActivityChooserModel dataModel = ActivityChooserModel.get(mContext, mShareHistoryFileName);
     dataModel.setOnChooseActivityListener(mOnChooseActivityListener);

